evil-jumps.el

```diff
@@ -156,9 +156,10 @@
                (pos (car place))
                (file-name (cadr place)))
           (setq evil--jumps-jumping t)
-          (if (string-match-p evil--jumps-buffer-targets file-name)
-              (switch-to-buffer file-name)
-            (find-file file-name))
+          (when (not (string= current-file-name file-name))
```

Comment:
`(unless (string= current-file-name file-name)`
tends to be the idiom

Reply:
Nice idea, thank you! Done!